Source/Core/Core/HW/DVD/DVDInterface.cpp

```diff
@@ -1222,6 +1222,8 @@ void ScheduleReads(u64 offset, u32 length, const DiscIO::Partition& partition, u
     // The last chunk may be short
     chunk_length = std::min(chunk_length, length);
 
+    // TODO: If the emulated software requests 0 bytes of data, should we seek or not?
```

Comment:
What was the old behavior? Did you run gc service disk against these changes?

Reply:
We were seeking before the chunking PR, and we seek with this PR. (With the chunking PR, we were essentially seeking except we were doing it too quickly, but it doesn't really matter since the game would hang anyway.)

I didn't run it before, but I ran the optical disc drive test now. It passes in master and in this PR, and it doesn't use any 0-length reads.

Reply:
I tested a bunch of the other weird edge-cases in Dolphin too.  This is the only game where I ran into a zero byte read.

Reply:
IIRC, Gecko OS does a zero byte read when launching a game. I haven't noticed anything else doing it, though.